TsRandomizer/Screens/Settings/GameSetting.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace TsRandomizer.Screens.Settings
+{
+	public abstract class GameSetting
+	{
+
+		[JsonProperty]
+		public dynamic CurrentValue { get; protected set; }
+		public string Name { get; set; }
```

Comment:
why are we not [JsonIgnore()] for the Name. Description and Default value as those values will be hardcoded in the code anyway